requirements/edx/github.txt

```diff
@@ -36,3 +36,4 @@ git+https://github.com/mitocw/django-cas.git@60a5b8e5a62e63e0d5d224a87f0b489201a
 -e git+https://github.com/edx/edx-oauth2-provider.git@0.4.0#egg=oauth2-provider
 -e git+https://github.com/edx/edx-val.git@ba00a5f2e0571e9a3f37d293a98efe4cbca850d5#egg=edx-val
 -e git+https://github.com/pmitros/RecommenderXBlock.git@b41ba8778b98da0ea680ffb8bbc59492d669df2d#egg=recommender-xblock
+-e git+https://github.com/edx-solutions/edx-search.git@021485b4b721ee49e6a5598ee14f02131b244990#egg=edx-search
```

Comment:
See https://github.com/edx-solutions/edx-search/pull/1 - will update this after review and creation of edx owned edx-search